### PR TITLE
Trimmed the OCS API toc to relevant information

### PR DESCRIPTION
**Pull request by mckenzielee**
- Removed SDS documentation from the OCS API landing page
- Consolidated the information in SDS landing page (Documentation/SequentialDataStore/Data_Store_and_SDS.md ) and the SDS introduction (Documentation/SequentialDataStore/Introducing_SDS.md)
- Removed Documentation/SequentialDataStore/Introducing_SDS.md 
- Removed information about support from Documentation/SequentialDataStore/Data_Store_and_SDS.md 
- Move documentation about architecture and accessing OCS to the OCS landing page Documentation/OSIsoft_Cloud_Services.md 
- Added a section about API versioning to the OCS landing page
- Updated the indexes and table of contents
- Removed the pictures on the OCS quick start page

### CODE REVIEW

File: Documentation/SequentialDataStore/Data_Store_and_SDS.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ uid: sds
 Sequential Data Store
 =====================
 
-The OSIsoft Cloud Services (OCS) Sequential Data Store (SDS) is an optimized sequential data store based on SDS technology.
+The OSIsoft Cloud Services (OCS) Sequential Data Store (SDS) is an optimized sequential data store based on SDS technology. SDS is used to store, retrieve, and analyze data. You create and write data to streams using a simple REST API. The streams are able to store simple or complex data types to suit the application needs. An assortment of methods with customizable behaviors are available to read data and easily obtain needed information.
 
 SDS is an OSIsoft technology that implements a data store that is optimized for dynamic data. Some of the features
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/Data_Store_and_SDS.md]: Before you made the changes, the content seemed too minimal to be it's own document... and even with your changes, it still seems incomplete.  E.g why are we only listing 2 features to describe SDS (and how did we choose those two?).  If this is intended to be a summary or introduction to what SDS is, I think we need more content.

That said, if we want to keep things as is, IANATS (I Am Not A Tom Seymour) but I would suggest a slight re-wording:

The OSIsoft Cloud Services (OCS) Sequential Data Store (SDS) is a hosted sequential data store optimized for dynamic data. SDS is used to store, retrieve, and analyze data. You create and write data to streams using a simple REST API. The streams are able to store simple or complex data types to suit the application needs. An assortment of methods with customizable behaviors are available to read data and easily obtain needed information. 
Some of the features of the data store include:
•	Support for complex data types
•	Support for complex indexes

Comment (mckenzielee) [reply Documentation/SequentialDataStore/Data_Store_and_SDS.md]: Good point. I will consolidate.

File: Documentation/OSIsoft_Cloud_Services.md [hunk 2]
```diff
@@ -13,3 +12,45 @@ The information in the sections that follow document the APIs and usage for each
...
 * Data ingress and OSIsoft Message Format - You use OSIsoft Message Format (OMF) to achieve high-throughput asynchronous 
   data ingress into the OCS Sequential Data Store. 
+
+
+## Access to OCS
+
+You must acquire a Tenant, security credentials, and a Namespace before interacting with the Data Store. If you do not 
+have a Tenant, you can request access to the OCS Preview at [OSIsoft Cloud Services](mailto://cloudservices@osisoft.com).
+
+The OCS Preview program is limited; contacting OSIsoft does not assure participation.
```
Comment (chad-at-osisoft) [inline Documentation/OSIsoft_Cloud_Services.md]: Should this preview stuff get scrubbed?

File: Documentation/OSIsoft_Cloud_Services.md [hunk 3]
```diff
@@ -13,3 +12,45 @@ The information in the sections that follow document the APIs and usage for each
...
+
+Isolation provided by Tenants and Namespaces prevents accidental or malicious data access. 
+
+## OCS API Versioning
+
+New versions of the API are released when major changes occur in the OCS API. When a newer version of the API is released 
+the legacy version of the API will only be supported for a period of time. We encourage all users to switch the 
+newer version as soon as possible. Versions are inline in the routes, following the host:
+
+    api/v{version}/Tenants/{tenantId}/Namespaces/{namespaceId}/...
```
Comment (chad-at-osisoft) [inline Documentation/OSIsoft_Cloud_Services.md]: Maybe this is a style thing, but putting "v" before the {version} locks us down to having versions that start with v. It doesn't do anything useful other than that.

File: Documentation/OSIsoft_Cloud_Services.md [hunk 4]
```diff
@@ -13,3 +12,45 @@ The information in the sections that follow document the APIs and usage for each
...
+New versions of the API are released when major changes occur in the OCS API. When a newer version of the API is released 
+the legacy version of the API will only be supported for a period of time. We encourage all users to switch the 
+newer version as soon as possible. Versions are inline in the routes, following the host:
+
+    api/v{version}/Tenants/{tenantId}/Namespaces/{namespaceId}/...  
+
+Where:  
+* version is an incrementally (singular) increasing integer value 
+
+* v{version}-preview will be used to expose beta functionality
```
Comment (chad-at-osisoft) [inline Documentation/OSIsoft_Cloud_Services.md]: I see now. The version string is more structured than I thought. I guess this is fine.

File: Documentation/support.md [hunk 5]
```diff
@@ -5,10 +5,8 @@ uid: support
 Getting support
 ===============
 
-The following email address is available to participants of the Qi
-Preview for both product support and feedback:
-
-[cloudservices@osisoft.com](mailto://cloudservices@osisoft.com)
+The following email address is available to participants of the OCS Preview
```
Comment (chad-at-osisoft) [inline Documentation/support.md]: Preview again.

File: Documentation/OSIsoft_Cloud_Services.md (hunk 2 above)
Comment (mckenzielee) [reply Documentation/OSIsoft_Cloud_Services.md]: I am glad you asked. I wasn't sure. I was hoping you, Laurent, and David could tell me what we should be saying in regards to availability.

File: Documentation/OSIsoft_Cloud_Services.md [hunk 6]
```diff
@@ -2,14 +2,55 @@
...
+Tenants are divided into one or more logical units called Namespaces. Each Namespace is distinct and separate from 
+other Namespaces, with its own instances of the various OSIsoft Cloud Services and data stores. 
+Namespaces provide isolation of resources. 
+
+Isolation provided by Tenants and Namespaces prevents accidental or malicious data access. 
+
+## OCS API Versioning
+
+New versions of the API are released when major changes occur in the OCS API. When a newer version of the API is released 
+the legacy version of the API will only be supported for a period of time. We encourage all users to switch the
```
Comment (osimarkrogers) [inline Documentation/OSIsoft_Cloud_Services.md]: Change to "encourage all users to switch to the new version..."

File: Documentation/OSIsoft_Cloud_Services.md [hunk 7]
```diff
@@ -2,14 +2,55 @@
...
+
+Tenants are divided into one or more logical units called Namespaces. Each Namespace is distinct and separate from 
+other Namespaces, with its own instances of the various OSIsoft Cloud Services and data stores. 
+Namespaces provide isolation of resources. 
+
+Isolation provided by Tenants and Namespaces prevents accidental or malicious data access. 
+
+## OCS API Versioning
+
+New versions of the API are released when major changes occur in the OCS API. When a newer version of the API is released
```
Comment (osimarkrogers) [inline Documentation/OSIsoft_Cloud_Services.md]: add comma after "released"

File: Documentation/OSIsoft_Cloud_Services.md (hunk 2 above)
Comment (mckenzielee) [reply Documentation/OSIsoft_Cloud_Services.md]: Laurent gave me some advice. I need you to review the changes.

File: Documentation/OSIsoft_Cloud_Services.md (hunk 4 above)
Comment (mckenzielee) [reply Documentation/OSIsoft_Cloud_Services.md]: I just copied from the API Guidelines. I am happy to change to anything that makes sense.

File: Documentation/SequentialDataStore/Quick_Start.md [hunk 8]
```diff
@@ -18,7 +16,7 @@ You will also need a Namespace and administrative client keys.
 
 #### Step 1: Acquire a Namespace
 
-Navigate to the OSIsoft Cloud Services page. Then, select the **Manage** tab and select **Namespaces**. For the 
+In the API Console, navigate to the OSIsoft Cloud Services page. Then, select the **Manage** tab and select **Namespaces**. For the
```
Comment (chad-at-osisoft) [inline Documentation/SequentialDataStore/Quick_Start.md]: In the OSIsoft Cloud Services portal, open the navigation menu and select Namespaces, which is in the Manage section. (Probably)

Comment (chad-at-osisoft) [reply Documentation/SequentialDataStore/Quick_Start.md]: Looking at it, there's no Manage section. It's under "Data Services"

Comment (osimarkrogers) [reply Documentation/SequentialDataStore/Quick_Start.md]: Yes, just Namespaces under Data Services.  The page, however, is called Manage Namespaces.

File: Documentation/OSIsoft_Cloud_Services.md [hunk 9]
```diff
@@ -2,14 +2,48 @@
...
-  data ingress into the OCS Sequential Data Store.
+* Sequential Data Store (SDS) - Use the information in this section to learn about the OSIsoft Cloud Services Sequential Data 
+  Store. The SDS is optimized for sequential data operations.
+* Data ingress and OSIsoft Message Format - You use OSIsoft Message Format (OMF) to achieve high-throughput asynchronous 
+  data ingress into the OCS Sequential Data Store. 
+
+
+## Access to OCS
+
+You must acquire an account before interacting with the OCS. If you do not
```
Comment (osimarkrogers) [inline Documentation/OSIsoft_Cloud_Services.md]: Just "OCS", delete "the"

**Pull request merged**